src/modules/fw_path_navigation/FixedwingPositionControl.cpp

```diff
@@ -2550,8 +2567,8 @@ FixedwingPositionControl::tecs_update_pitch_throttle(const float control_interva
 	}
 
 	/* update TECS vehicle state estimates */
-	const float throttle_trim_comp = compensateTrimThrottleForDensityAndWeight(_param_fw_thr_trim.get(), throttle_min,
-					 throttle_max);
+	const float throttle_trim_comp = calculateTrimThrottle(throttle_min,
```

Comment:
maybe keep the name in sync with the internal state in TECS - `throttle_trim_adjusted`

Reply:
3111a6b362afe19dc1488d5e91ca2a297466be46